modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase

```diff
@@ -413,23 +415,24 @@ definition {
 
 		PageEditor.clickPublish();
 
-		User.logoutPG();
+		var portalURL = PropsUtil.get("portal.url");
```

Comment:
This is ok for now but in the future we should try to put this into a macro. I've seen this a few times